doc/spec.adoc

```diff
@@ -55,6 +59,28 @@ This field can be used for layered products to enforce their dependencies so tha
 Each element is composed of:
 ** Mandatory `groupId` and `artifactId` elements that are the Maven coordinates of the required channel.
 ** Optional `version` to stick to a given channel version (instead of requiring the latest version of that channel). In the absence of this `version`, the latest version of the channel will be determined based on the Maven repository metadata.
+* A collection of `repositories` that defines repositories associated with the channel. Only listed repositories are used to resolve channel's components. Each repository is composed of:
+** A required `id` that defines the repository name used in Maven cache
+** A required `url` pointing to a default Maven repository
+* Optional `manifest` corresponding to the Channel Manifest artefact. Channel Manifest is used to define streams available in the channel.
+** One of the following, mutually exclusive fields, used to resolve the Channel Manifest:
+*** `gav` corresponds to Maven coordinates of the Manifest. It can either use format of `groupId:artifactId` (resulting in the latest available version being used) or `groupId:artifactId:version` (resulting in concrete version being used)
+*** `url` corresponding to a static URL where the manifest file can be found.
+* Optional `blocklist` corresponding to the Blocklist artefact. Blocklist is used to define versions of artifacts excluded from a channel.
+** One of the following, mutually exclusive fields, used to resolve the Blocklist:
+*** `gav` corresponds to Maven coordinates of the Manifest. It can either use format of `groupId:artifactId` (resulting in the latest available version being used) or `groupId:artifactId:version` (resulting in concrete version being used)
+*** `url` corresponding to a static URL where the manifest file can be found.
+* An optional `resolves-if-no-stream` that defines strategy for version resolution if the artifact has not been found in the `streams` collection. Allowed values are:
+** `original` - fallback to the `baseVersion` provided when querying the channel. Default value if no strategy is provided.
+** `latest` - use the latest version found in the Maven repositories
+** `maven-latest` - a version marked as `latest` in the Maven metadata
+** `maven-release` - a version marked as `release` in the Maven metadata
+** `none` - do not attempt to resolve versions of artifact not listed in the `streams` collection
+
+A Channel Manifest is composed of following fields:
```

Comment:
Can you add subheading for the 3 definitions, this will make it easier to reference them separately if needs be

Reply:
Added missing blocklist definition below as well